envoy/datadog_checks/envoy/metrics.py

```diff
@@ -108,7 +108,6 @@
     'envoy_cluster_upstream_rq_rx_reset': 'cluster.upstream_rq_rx_reset',
     'envoy_cluster_upstream_rq_time': 'cluster.upstream_rq_time',
     'envoy_cluster_upstream_rq_timeout': 'cluster.upstream_rq_timeout',
-    'envoy_cluster_upstream_rq': 'cluster.upstream_rq',
```

Comment:
A bit surprised by this one, I would expect some linting rule to catch this (duplicate dict key) 🤔 .

Reply:
To be honest, I missed that one too, my IDE linter caught it 😅 

I took a look and it seems our flake8 catches F601 (dictionary key name repeated with different values) but not F602 (dictionary key variable name repeated with different values). I'll try to find out why 🙂